### DISCUSSION TITLE
TypeError: fetch failed

### DISCUSSION THREAD

**Discussion by psebestyen** [blefnk/relivator#8] [Q&A (questions and answers)]
Hello,

I really like this project! There is a lot I can learn from this as I am new to nextjs. I am able to build and start the project, but when I try to pull it up in the browser I get the following error:
![image](https://github.com/blefnk/relivator/assets/3189002/97d02173-5600-4547-aeeb-13ac753f42df)
I have set up clerk, the db connection and the next auth secret, but nothing else. I am running it on Windows. Any ideas what might be wrong? I would really like to run this in addition to just being able to examine the code.

Thanks

**Comment by blefnk** [comment]
Hello and welcome 🎉! I've fixed this bug in the 1.0.0 version. It will be published later today or tomorrow. Thank you for taking an interest in this project!

**Reply by Yazan-Ali-01** [reply]
hey again!
It's heppening with me rn (20/4/2024)
I've installed the packages then ran mysql db container with: 
`docker run -p 3306:3306 --name testing-mysql -e MYSQL_ROOT_PASSWORD=my-secret-pw -d mysql:latest`
and I made sure that it's running and working 
then filled env variables as the following :
```
DATABASE_URL="mysql://root:my-secret-pw@localhost/mysql"
NEXT_PUBLIC_DB_PROVIDER="planetscale"
```
then trying now to to do `pnpm mysql:push` but getting 
`  const createConnectionErr = new Error();
                                  ^

Error
    at createConnection2 (C:\Users\FPCC\Desktop\WEB DEV\relivator-nextjs-template\node_modules\.pnpm\drizzle-kit@0.20.14\node_modules\drizzle-kit\bin.cjs:44322:35)
    at connectToMySQL (C:\Users\FPCC\Desktop\WEB DEV\relivator-nextjs-template\node_modules\.pnpm\drizzle-kit@0.20.14\node_modules\drizzle-kit\bin.cjs:61903:64)
    at Command.<anonymous> (C:\Users\FPCC\Desktop\WEB DEV\relivator-nextjs-template\node_modules\.pnpm\drizzle-kit@0.20.14\node_modules\drizzle-kit\bin.cjs:63065:28) {
  code: 'ECONNREFUSED',
  errno: undefined,
  sqlState: undefined
}

Node.js v21.4.0
 ELIFECYCLE  Command failed with exit code 1.`
 
 and if i try `pnpm dev` I get the same error (Fetch Failed) 
 Please help as I was impressed by the project effort and really want to contribute

**Comment by psebestyen** [comment]
That's great news. Thanks. I will check it out.

**Comment by blefnk** [comment]
@psebestyen ✅ [**v1.0.0 🎉 Relivator Release**](https://github.com/blefnk/relivator/releases/tag/1.0.0)
